nixos/modules/services/x11/xserver.nix

```diff
@@ -14,8 +14,8 @@ let
   # Map video driver names to driver packages. FIXME: move into card-specific modules.
   knownVideoDrivers = {
     virtualbox = { modules = [ kernelPackages.virtualboxGuestAdditions ]; driverName = "vboxvideo"; };
-    ati = { modules = with pkgs.xorg; [ xf86videoati glamoregl ]; };
-    intel = { modules = with pkgs.xorg; [ xf86videointel glamoregl ]; };
+    ati = { modules = with pkgs.xorg; [ xf86videoati ]; };
+    intel = { modules = with pkgs.xorg; [ xf86videointel ]; };
```

Comment:
Now we can completely remove those two lines. The mapping to `xf86videoXXX` is automatic (around line 446).